ConnectorAutocadCivil/ConnectorAutocadCivil/UI/ConnectorBindingsAutocadCivil.Receive.cs

```diff
@@ -217,20 +220,30 @@ string id
             return;
           }
 
-          // convert base (or base fallback values) and store in appobj converted prop
-          if (commitObj.Convertible)
+          if (StoredObjects.TryGetValue(commitObj.OriginalId, out Base commitBaseObj))
           {
-            if (StoredObjects.TryGetValue(commitObj.OriginalId, out Base obj))
+            // log received object type
+            if (typeCountDict.TryGetValue(commitBaseObj.speckle_type, out int value))
             {
-              converter.Report.Log(commitObj); // Log object so converter can access
-              commitObj.Converted = ConvertObject(obj, converter);
+              typeCountDict[commitBaseObj.speckle_type] = ++value;
             }
             else
             {
-              commitObj.Update(status: ApplicationObject.State.Failed, logItem: "Object not found in StoredObjects");
+              typeCountDict.Add(commitBaseObj.speckle_type, 1);
```

Comment:
This can be simplified, no need for branching:
typeCountDict.TryGetValue(commitBaseObj.speckle_type, out int value); 
typeCountDict[commitBaseObj.speckle_type] = ++value;